src/resmom/start_exec.c

```diff
@@ -2637,13 +2688,26 @@ receive_pipe_request(int sd)
 	(void)writepipe(pjob->ji_mjspipe2, &cmd, sizeof(int));
 
 	if (cmd == IM_EXEC_PROLOGUE) {
+
 		if (send_sisters(pjob, IM_EXEC_PROLOGUE, NULL) != pjob->ji_numnodes - 1) {
 			snprintf(log_buffer, sizeof(log_buffer),
 			  	"warning: %s: IM_EXEC_PROLOGUE requests "
 			  	"could not reach some sister moms",
 				pjob->ji_qs.ji_jobid);
 			log_err(-1, __func__, log_buffer);
 		}
+
+		if (do_tolerate_node_failures(pjob)) {
+			long delay_value;
+			/* execute report task 'delay_value' seconds from
+			 * now, where the value is 95 % of the job_launch_delay
+			 * value. This allows the waiting child mom to
+			 * capture the failed node host values before it times
+			 * out on job_launch_delay.
+			 */
+ 			delay_value = 0.95 * job_launch_delay;
+			(void)set_task(WORK_Timed, time_now + delay_value, report_failed_node_hosts_task, pjob);
```

Comment:
The number of times report_failed_node_hosts_task get "spinned" is not dependent on the job_launch_delay value. It will actually just be 1 timed task per job if there's prologue hook (or hooks).